src/server/server.h

```diff
@@ -341,6 +343,8 @@ class Server {
   std::mutex last_random_key_cursor_mu_;
 
   std::atomic<lua_State *> lua_;
+  // The cache of flag is cached when the script is created and cleared when the script is flushed.
```

Comment:
Maybe we should denote this is sha -> Flag, and changing value to `ScriptFlags`?

Reply:
Although this area may need to be redesigned, it is worth thinking about. 

I think we need to rename `ScriptFlags=>ScriptFlagType` and add `using ScriptFlags = uint64_t;`.
`ScriptFlags` is composed of `ScriptFlagType` using bit operations. This is more clear. What do you think?

Reply:
LGTM